core/vm/evm.go

```diff
@@ -140,24 +140,21 @@ func NewEVM(ctx Context, statedb StateDB, chainConfig *params.ChainConfig, vmCon
 	}
```

Comment:
`make([]Interpreter, 0, 1),` -> `make([]Interpreter, 0, 2),` to cater for eventually having both eWASM and EVM